src/components/filterPanel/FilterPanel.js

```diff
@@ -360,31 +434,38 @@ const FilterPanel = ({
           )}
         />
       </div>
-      <div className={styles.FilterContainer}>
-        {renderOptions()}
-
-        {optionValue !== ContextMenuOptions.ALL ? <h5>Jaar</h5> : ''}
-        {optionValue === ContextMenuOptions.BLACKSPOTS
-          ? renderBlackspotYearCheckboxes()
-          : ''}
-
-        {optionValue === ContextMenuOptions.DELIVERED
-          ? renderDeliveredYearCheckboxes()
-          : ''}
-
-        {optionValue === ContextMenuOptions.QUICKSCANS
-          ? renderQuickscanYearCheckboxes()
-          : ''}
-
-        {optionValue === ContextMenuOptions.ALL ||
-        optionValue === ContextMenuOptions.DELIVERED
-          ? renderTypeCheckboxes()
-          : ''}
-
-        {optionValue !== ContextMenuOptions.DELIVERED
-          ? renderStatusCheckboxes()
-          : ''}
-      </div>
+      <FilterWrapperStyle>
+        <div className={styles.FilterContainer}>
+          {renderOptions()}
+          {optionValue !== ContextMenuOptions.ALL ? <h5>Jaar</h5> : ''}
```

Comment:
Conditional rendering should return a component or `undefined` or `null`.
You can write
```jsx
{optionValue !== ContextMenuOptions.ALL && <h5>Jaar</h5>}
```
Also goes for each occurrence below

Reply:
done!